### DISCUSSION TITLE
Exclude some content from search summary

### DISCUSSION THREAD

**Discussion by El-Gavy** [squidfunk/mkdocs-material#4480] [Q&A] [answered]
Hello,

First of all, thanks a lot for your work, Material for MkDocs is an awesome project!

My question : I maintain an unofficial translation for [Zotero](https://zotero.org) : https://docs.zotero-fr.org

As it is an unofficial translation, to credit the official documentation all my pages begins with a link to the original resource : 

![image](https://user-images.githubusercontent.com/26960332/194815184-12bd706a-33d0-4e60-815a-3e86c7ac3cf9.png)

My problem is that this text is logically used in the search summary, which is not relevant : 

![image](https://user-images.githubusercontent.com/26960332/194815701-7a6ab953-0d8d-442f-9cde-d50fd16016a6.png)

Is there a way to "exclude" some content from being used the search summary?

**Comment by squidfunk** [comment, marked as answer]
Thanks! Yes, there is, [and it's super simple](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#excluding-blocks):

``` markdown
# Document title

The content of this block is included

The content of this block is excluded
{ data-search-exclude }
```

Note that this feature is [currently reserved for sponsors](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#search-exclusion), as it's solely available via [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/). If you decide to sponsors, you can use it immediately. Otherwise, you have to wait until the funding goal to which it is tied is reached.

**Reply by El-Gavy** [reply]
Thanks for your quick answer! That’s indeed very simple and userfriendly, great implementation.
 We are an unformal non-profit collective of librarians so we have no dedicated funds for the moment but we will discuss it :)
